k6/src/typescript/hoprd-node.ts

```
@@ -51,91 +53,222 @@ export class HoprdNode {
+          sent = true;
+          console.log(`[INFO] Route sender=${sender}, relayer=${relayer.data.name}, receiver=${receiver.data.name} works`);
+          resolve(true);
+        } else {
+          console.error(`Message failed, sender=${sender}, relayer=${relayer.data.name}, receiver=${receiver.data.name}`);
+          resolve(false);
+        }
+        ws.close(); // Close connection after receiving the response
+      };
+
+      // Event: Error
+      ws.onerror = function error(error: any) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'error'. Consider to delete it or rename it.

'error' is defined here:


(lint/suspicious/noRedeclare)